AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -464,9 +464,8 @@ class CallingContext: NSObject {
     }
 }
 
-extension CallingContext: CallDelegate,
-                          RecordingFeatureDelegate,
-                          TranscriptionFeatureDelegate {
+extension CallingContext: CallDelegate, RecordingCallFeatureDelegate,
+                            TranscriptionCallFeatureDelegate {
```

Comment:
Nit: separate each delegates into a newline like before

Reply:
@cheukchuen Thanks for the pointer, updated PR